hi.js

```diff
@@ -902,15 +902,15 @@ settings.quality.flapAnim स्नाइपर फ्लैप एनीमे
 chall.name.hs Get {0} हैडशॉट्स {1}
 chall.name.ns Get {0} नो-स्कोप्स {1}
 chall.name.ls Get {0} लेगशॉट्स {1}
-chall.name.tmk Get {0} throwing knife kills {1}
-chall.name.fb Get {0} first bloods {1}
+chall.name.tmk Get {0} चाक़ू फेंकने से किल्स {1}
+chall.name.fb Get {0} पहला खून {1}
 chall.name.mk Get {0} मिली किलज़ {1}
 chall.name.fk Get {0} हाँथों द्वारा किये गए किल {1}
 chall.name.qs Get {0} क्विक-स्कोप्स {1}
-chall.name.wb Get {0} wallbangs {1}
-chall.name.airk Get {0} mid-air kills {1}
-chall.name.lns Get {0} longshots {1}
-chall.name.tk Get {0} turbo kills {1}
+chall.name.wb Get {0} वालबैंग्स {1}
+chall.name.airk Get {0} हवा में किल्स {1}
+chall.name.lns Get {0} दूर के किल्स {1}
+chall.name.tk Get {0} तेज़ किल्स {1}
```

Comment:
टर्बो किल would be a better alternative

Reply:
maybe

Reply:
ye turbo kill makes sense tej kill kinda wierd